src/CAREUI/misc/PrintPreview.tsx

```
@@ -12,24 +12,35 @@ type Props = {
-      <div className="mx-auto my-8 w-[50rem]">
-        <div className="top-0 z-20 flex gap-2 bg-secondary-100 px-2 py-4 xl:absolute xl:right-6 xl:top-8 xl:justify-end">
+      <div className="mx-auto my-8 max-w-full px-4 lg:w-[50rem]">
+        <div className="mb-3 flex flex-wrap justify-end gap-2">
           <ButtonV2 disabled={props.disabled} onClick={print}>
             <CareIcon icon="l-print" className="text-lg" />
             {t("print")}
           </ButtonV2>
+          {props.onClose && (
+            <ButtonV2
+              variant="secondary"
+              onClick={() => props.closeFeature && props.closeFeature(false)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)